litigious-liberators/profile_cards/profile_cards.py

```diff
@@ -0,0 +1,210 @@
+from kivymd.toast import toast
+from kivy.uix.behaviors import ButtonBehavior
+from kivy.uix.image import Image
+from kivy.config import Config
+from kivy.app import App
+import os
+from os import listdir
+from random import shuffle
+from copy import deepcopy
+from kivy.uix.screenmanager import ScreenManager, Screen, SlideTransition
+from kivy.uix.popup import Popup
+from kivy.uix.button import Button
+from kivy.uix.scrollview import ScrollView
+from kivy.uix.boxlayout import BoxLayout
+from kivy.properties import StringProperty, DictProperty
+from yaml import safe_load
+from kivy.core.window import Window
+from collections import Counter
+from kivy.uix.label import Label
+from kivy.core.audio import SoundLoader
+
+Config.set("input", "mouse", "mouse,multitouch_on_demand")
+
+
+class ScrollableLabel(ScrollView):
+    text = StringProperty("")
+
+
+class Content(BoxLayout):
+    def __init__(self, **kwargs):
+        super(Content, self).__init__(**kwargs)
+        self.orientation = "vertical"
+        self.story = ScrollableLabel()
+        self.button = Button(text="Close")
+        self.add_widget(self.story)
+        self.add_widget(self.button)
+
+
+class ImageButton(ButtonBehavior, Image):
+    def __init__(self, **kwargs):
+        super(ImageButton, self).__init__(**kwargs)
```

Comment:
I saw this comment in many code jam reviews, and I think the biggest reason `super(cls, self)` is used all over the places is because it is what we all see in kivy docs :)

Reply:
i did think that too, not too long ago kivy supported python2 (and latest release still does), so the doc use the syntax that works in both versions.